### DISCUSSION TITLE
cross-spawn dependency vulnerability

### DISCUSSION THREAD

**Discussion by manuweg** [yakovmeister/pdf2image#218] [Q&A]
will you be updating this any time soon?

`# npm audit report

cross-spawn  <6.0.6 || >=7.0.0 <7.0.5
Severity: high
Regular Expression Denial of Service (ReDoS) in cross-spawn - https://github.com/advisories/GHSA-3xgq-45jj-v275
Regular Expression Denial of Service (ReDoS) in cross-spawn - https://github.com/advisories/GHSA-3xgq-45jj-v275
No fix available
node_modules/cross-spawn
node_modules/gm/node_modules/cross-spawn
  gm  >=1.23.0
  Depends on vulnerable versions of cross-spawn
  node_modules/gm
    pdf2pic  *
    Depends on vulnerable versions of gm
    node_modules/pdf2pic`

**Comment by mskec** [comment]
Hi @manuweg, the vulnerable package is used by pdf2pic `gm` dependency. 
There's an open issue in their repository https://github.com/aheckmann/gm/issues/873
